tests/scenarios/executors/gotest.py

```diff
@@ -0,0 +1,44 @@
+import os
+import unittest.mock
+from typing import cast
+
+from pytest_mock import MockerFixture
+from step_exec_lib.types import StepType
+
+import app_test_suite
+import app_test_suite.steps.executors.gotest
+from tests.helpers import MOCK_KUBE_VERSION
+
+
+def assert_run_gotest(test_provided: StepType, kube_config_path: str, chart_file: str, app_version: str) -> None:
+    env_vars = {
+        "ATS_APP_CONFIG_FILE_PATH": "",
+        "ATS_CHART_PATH": chart_file,
+        "ATS_CHART_VERSION": app_version,
+        "ATS_CLUSTER_TYPE": "mock",
+        "ATS_CLUSTER_VERSION": MOCK_KUBE_VERSION,
+        "ATS_KUBE_CONFIG_PATH": kube_config_path,
+        "ATS_TEST_TYPE": test_provided,
+        "ATS_TEST_DIR": "",
+        "GOPATH": os.getenv("GOPATH", ""),
+        "HOME": os.getenv("HOME", ""),
+        "PATH": os.getenv("PATH", ""),
+    }
+
+    # Set env vars needed for Go.
```

Comment:
This comment doesn't make sense any more. Can we this the same as in the go executor for consistency?

https://github.com/giantswarm/app-test-suite/blob/62634bbd5f3d3676fafe86c5e37344cdb7bf4807/app_test_suite/steps/executors/gotest.py#L75-L78